Gemfile

```diff
@@ -22,7 +22,7 @@ gem 'hiredis'
 gem 'http_accept_language'
 gem 'httparty'
 gem 'identity-hostdata', github: '18F/identity-hostdata', branch: 'master'
-gem 'identity-telephony', github: '18f/identity-telephony', tag: 'v0.0.16'
+gem 'identity-telephony', github: '18f/identity-telephony', branch: 'jmhooper-return-response-from-senders'
```

Comment:
I need to come back and replace this with the latest version of the gem. That will have to wait until we upgrade